deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -208,3 +215,9 @@ def condition(result: Dict) -> ConditionResult:
         return self.add_condition(f'PSI <= {max_allowed_psi_score} and Earth Mover\'s Distance <= '
                                   f'{max_allowed_earth_movers_score} for label drift',
                                   condition)
+
+
+def get_column_type(output_type):
```

Comment:
Shouldn't this be in some central location?

Reply:
done